addons/website_event_exhibitor/views/event_exhibitor_templates_list.xml

```diff
@@ -121,13 +121,18 @@
     <!-- No exhibitors -->
     <t t-if="not sponsor_categories">
         <div class="col-12">
-            <div class="h2 mb-3">No exhibitor found.</div>
-            <div t-if="search_key" class="alert alert-info text-center">
-                <p class="m-0">We did not find any exhibitor matching your <strong t-out="search_key"/> search.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
```

Comment:
Here again, flex classes are probably unnecessary ... could you check all the occurrences?